tools/syz-trace2syz/config/constants.go

```diff
@@ -0,0 +1,36 @@
+package config
+
+var (
+	//Strace uses its own flag names. Map them to real ones
+	Consts = map[string]string{
+		"O_ASYNC": "FASYNC", //O_ASYNC is used in libc headers
+
+	}
+	// SpecialConsts contains values for flags found in strace. Some of these have different names in Syzkaller
+	// and others are not yet added to Syzkaller.
+	SpecialConsts = map[string]uint64{
+		"SIG_0": 0,
+		"BUS":   10,
+		"USR1":  30,
```

Comment:
Yes this is a mistake. I will move BUS, USR1, IO, ALRM to Consts map. The R_OK, W_OK  flags are just for the ```access``` system call which is not in supported by Syzkaller. Is there a reason you don't support ```access``` or should I add the its description along with these flags?

Reply:
I suspect we don't have it because it's effectively the same as faccessat that we have:

```
SYSCALL_DEFINE3(faccessat, int, dfd, const char __user *, filename, int, mode)
{
	return do_faccessat(dfd, filename, mode);
}

SYSCALL_DEFINE2(access, const char __user *, filename, int, mode)
{
	return do_faccessat(AT_FDCWD, filename, mode);
}
```

So there is probably not much value in adding it separately. And there is actually some negative effect of adding it: arm64 kernels generally have only *at versions of syscalls and make libc emulate the non-*at versions via *at versions. So if syzkaller finds a crash with access call on amd64, it will not be possible to reuse the program on arm64.